pkgs/swiftgen/swift2objc/analysis_options.yaml

```diff
@@ -2,23 +2,6 @@ include: package:dart_flutter_team_lints/analysis_options.yaml
 
 analyzer:
   language:
-    strict-casts: false # TODO(https://github.com/dart-lang/native/issues/1278): Fix.
```

Comment:
I think these 3 default to false, and need to be explicitly set to true, rather than just deleted.

```yaml
analyzer:
  language:
    strict-casts: true
    strict-inference: true
    strict-raw-types: true
```

Reply:
added